spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :request do
```

Comment:
Awesome! You used type: request as expected 👍

Reply:
`index` and `show` actions for `User` and `Post` controllers are tested 🔥

Reply:
Thank you so much for the positive feedback! 


![than k_you](https://user-images.githubusercontent.com/48417586/191326796-7b3b0f7e-ef68-4f00-9fb8-c4edb4db18e5.gif)